### DISCUSSION TITLE
I can't apply for a student discount

### DISCUSSION THREAD

**Discussion by scrwnl** [community/community#52526] [GitHub Education]
Many times I sent an image of my student ID written in Japanese with an English translation, but it was immediately rejected.
What should I do?
![IMG_1448](https://user-images.githubusercontent.com/63852665/231189005-9638c652-9e15-4bf0-b2b4-9e8a825abd5e.jpeg)

**Comment by samus-aran** [comment]
Hi @scrwnl 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍

**Reply by scrwnl** [reply]
Thank you for your reply.
A while ago, I did what you said, but It was not approved. What else am I missing?
![IMG_1458](https://user-images.githubusercontent.com/63852665/232083358-8928ab27-0593-4128-9722-11ecb6c61f03.jpeg)
